source/elements/oneMKL/source/domains/rng/device_routines/device-engines.rst

```diff
@@ -0,0 +1,42 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_engines:
+
+Device Engines (Basic Random Number Generators)
+===============================================
+
+oneMKL RNG provides two device pseudorandom number generators:
+
+.. tabularcolumns:: |\Y{0.4}|\Y{0.6}|
+
+.. list-table::
+   :header-rows: 1
+   :class: longtable
+
+   * -  Routine
+     -  Description
+
+   * -  :ref:`onemkl_device_rng_mrg32k3a`
+     -  The combined multiple recursive pseudorandom number generator ``MRG32k3a`` [:ref:`L'Ecuyer99 <onemkl_device_rng_bibliography>`]
+
+   * -  :ref:`onemkl_device_rng_philox4x32x10`
+     -  Philox4x32-10 counter-based pseudorandom number generator with a period of 2 :sup:`128` ``PHILOX4X32X10`` [:ref:`Salmon11 <onemkl_device_rng_bibliography>`]
```

Comment:
2 :sup:`128` looks like formula. Please, wrap it to `math`

Reply:
changed, thanks